pkg/csi-common/controllerserver-default.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
```

Comment:
i would like to keep kuberentes as the author for all files inside the csi-common folder, these  files are taken from kubernetes-csi/driver repo